fields/mixins/ArrayField.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
 	},
 
 	componentWillReceiveProps: function (nextProps) {
+		if (!nextProps.value) return;
```

Comment:
If nextProps.value doesn't exist (or is generally falsey) should we clear the component for the user? ie. `this.setState({ values: null });` or even `this.setState({ values: [] });` ?

Reply:
Was this issue resolved in keystone 4.0.0 . I am facing the issue if there is no value passed